src/ui/setup/dynamic/devtools.ts

```diff
@@ -80,12 +80,11 @@ enum SessionError {
 const SessionErrorMessages: Record<number, string> = {
   [SessionError.BackendDeploy]: "Please wait a few minutes and try again.",
   [SessionError.NodeTerminated]: "Our servers hiccuped but things should be back to normal soon.",
-  [SessionError.KnownFatalError]:
-    "This error has been fixed in an updated version of Replay. Please try upgrading Replay and trying a new recording.",
+  [SessionError.KnownFatalError]: "Darn! There was an error. Please try recording again.",
```

Comment:
This doesn't look right to me.  It sounds good to have the same error message for KnownFatalError and OldBuild like we did before, but this PR is updating the message for KnownFatalError and not UnknownFatalError.  I think this should leave the message for KnownFatalError alone, and use this new message for UnknownFatalError.

Reply:
Good catch, will fix now

Reply:
Moving to new PR: https://github.com/RecordReplay/devtools/pull/5907